.github/workflows/build_push_ghcr.yaml

```diff
@@ -2,7 +2,7 @@ name: build image and push to ghcr.io
 
 on: 
   push:
-    branches: [main-ls2]
+    branches: ["main-ls2", "sprint"]
```

Comment:
Der Job wird bei nun zusätzlich bei einem Push in den `sprint`-Branch ausgelöst.

Wird das bisherige Image erstellt durch einen Push in den Branch `main-ls2` überschrieben? Falls ja, ist nicht ersichtlich auf welchem Branch das Image erstellt wurde.

Reply:
> Wird das bisherige Image erstellt durch einen Push in den Branch main-ls2 überschrieben?

ne weil er den tag variable holt 

`          tags: ${{ steps.meta.outputs.tags }}`